caffe2/ideep/operators/elementwise_sum_op.cc

```diff
@@ -12,16 +12,23 @@ class IDEEPSumOp final : public IDEEPOperator {
   virtual ~IDEEPSumOp() {}
 
   bool RunOnDevice() override {
-    const auto &X = Input(INPUT0);
+    const auto& X = Input(INPUT0);
     auto* Y = Output(OUTPUT);
 
     if (InputSize() == 1) {
       ideep::direct_copy::compute(X, *Y);
```

Comment:
does it handle inplace?

Reply:
You mean what if `Y == X`? Good question. Let's ask Intel folks. How do we check Inplace? Just by their blob name?

Reply:
You can check by comparing blob pointers such as `if (&X == Y)`. IDEEP does appear to support for certain ops, but it's work checking how sum is handled.